src/component/base/DraftEditor.react.js

```diff
@@ -302,9 +302,21 @@ class DraftEditor extends React.Component<DraftEditorProps, State> {
 
       return <DraftEditorPlaceholder {...placeHolderProps} />;
     }
+    this._placeholderAccessibilityID = null;
     return null;
   }
 
+  _getAriaDescribedByIDs(): ?string {
+    const ariaDescribedByIDs = [
+      this._placeholderAccessibilityID,
+      this.props.ariaDescribedBy,
```

Comment:
Without type safety, I'm concerned that folks might pass in either a string or an Array for this prop value. Is there a way to guarantee a string? Or, if not, you should run this prop through `Array.isArray` and spread it (or join it) before joining with the placeholder id.

Reply:
I understand the concern. On the other hand, should this apply also to other props, for example `ariaLabelledBy`? I couldn't find in the codebase other examples of checking for a prop value this way, and it would look a bit overkill to me. If you strongly feel it's necessary, please do let me know.